vtr_flow/arch/titan/stratixiv_arch.timing.xml

```diff
@@ -4408,6 +4408,14 @@
         <port name="vcounderrange" clock="inclk"/>
       </output_ports>
     </model>
+    <model name="stratixiv_clkena">
```

Comment:
Useful to comment what this is: the clkena block at the root of block trees.

Reply:
Thank you, Vaughn, for catching these issues. I have made the changes as recommended.